apps/rush-lib/src/api/RushConfigurationProject.ts

```diff
@@ -358,10 +389,13 @@ export class RushConfigurationProject {
     return isMain;
   }
 
-  private _getLocalDependencyProjects(
+  /**
+   * Compute the local rush projects that this project immediately depends on,
+   * according to the specific dependency group from package.json
+   */
+  private *_getLocalDependencyProjects(
```

Comment:
Just return an array here.

Reply:
But the generator is actually less code, unlike the other scenarios you called out

Reply:
Switched to a `Set<RushConfigurationProject>` like for its counterpart.